src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,16 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
-    /**
-     * Imagine that some user wants to login to your site.
-     * You should check if user credentials (login and password) are valid or not.
-     * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
-     * @param password - user's password
-     * @return true if user by email exists and passed password is equal to user's password.
-     * Return false in any other cases.
-     */
     public boolean login(String email, String password) {
+        UserService userIsNotNull = new UserService();
+        User currentUser = new UserService().findByEmail(email);
+        if (userIsNotNull.findByEmail(email) != null
+                && userIsNotNull.findByEmail(email).getEmail().equals(email)
+                && currentUser.getPassword().equals(password)) {
+            return true;
+        }
         return false;
```

Comment:
let's improve naming and don't repeat method calls when we can use stored result
also don't create another object of user service when you can use existing one